docs/BUILDING.md

```diff
@@ -2,31 +2,67 @@
 
 ## Prerequisites
 
-[JavaSE-11](https://www.oracle.com/ca-en/java/technologies/javase-jdk11-downloads.html), [Maven](https://maven.apache.org/download.cgi), and [Eclipse](https://www.eclipse.org/downloads/) are required to build `lsp4jakarta` and `jakarta-eclipse`.
+[JavaSE-11](https://www.oracle.com/ca-en/java/technologies/javase-jdk11-downloads.html), [Maven](https://maven.apache.org/download.cgi), and [Eclipse](https://www.eclipse.org/downloads/) (Eclipse IDE for Enterprise Java and Web Developers is recommended) are required to build the lsp4jakarta project.
```

Comment:
For clarity, are these requirements for this entire LSP4Jakarta project (i.e. all three of jakarta.ls, jakarta.jdt, jakarta.eclipse)?

Reply:
Right now, the only way to run the project is through Eclipse so users would need all of these for all three components